distributions/openhab/src/main/resources/conf/services/runtime.cfg

```diff
@@ -46,6 +66,107 @@
 #
 #org.openhab.voice:defaultHLI=
 
+# The default keyword spotting service to use if no other is specified.
+#
+#org.openhab.voice:defaultKS=
+
+# The magic word to spot before initiating a dialog.
+#
+#org.openhab.voice:keyword=
+
+# If provided, the item will be switched on during the period when the dialog processor has spotted the keyword and is listening for commands.
+#
+#org.openhab.voice:listeningItem=
+
+
+#################### UI ####################
+
+# Defines the UI theme.
+#
+#org.openhab.basicui:theme=default
+
+# Defines whether UI renders icons for the widgets or not.
+#
+#org.openhab.basicui:enableIcons=true
+
+# When enabled, changes the layout so that more widgets can fit on the screen.
+#
+#org.openhab.basicui:condensedLayout=false
+
+# Displays all widget states in uppercase.
+#
+#org.openhab.basicui:capitalizeValues=false
+
+# Play audio sent to the web audio sink.
+#
+#org.openhab.basicui:webAudio=false
+
+# The sitemap to show if no parameter is passed.
+#
+#org.openhab.basicui:defaultSitemap=
+
+
```

Comment:
Maybe it would better to create `basicui.cfg` file for this in the webui project. I.e. similar to how configuration files are installed for other add-ons using the openhab-addons-external feature:

https://github.com/openhab/openhab-addons/tree/main/features/openhab-addons-external/src/main/resources/conf